src/System.Management.Automation/security/nativeMethods.cs

```diff
@@ -1088,36 +1088,36 @@ internal static DWORD DestroyWintrustDataStruct(WINTRUST_DATA wtd)
         [StructLayout(LayoutKind.Sequential)]
         internal struct CRYPT_PROVIDER_CERT
         {
-            private DWORD _cbStruct;
+            private readonly DWORD _cbStruct;
             internal IntPtr pCert; // PCCERT_CONTEXT
-            private BOOL _fCommercial;
-            private BOOL _fTrustedRoot;
-            private BOOL _fSelfSigned;
-            private BOOL _fTestCert;
-            private DWORD _dwRevokedReason;
-            private DWORD _dwConfidence;
-            private DWORD _dwError;
-            private IntPtr _pTrustListContext; // CTL_CONTEXT*
-            private BOOL _fTrustListSignerCert;
-            private IntPtr _pCtlContext; // PCCTL_CONTEXT
-            private DWORD _dwCtlError;
-            private BOOL _fIsCyclic;
-            private IntPtr _pChainElement; // PCERT_CHAIN_ELEMENT
+            private readonly BOOL _fCommercial;
```

Comment:
It seems we could make whole struct `readonly`.

Reply:
I am not sure, we should be careful with pinvoke code. We could consider this in #13919.

Reply:
See issue #13915.